src/string_bytes.cc

```diff
@@ -72,10 +73,15 @@ class ExternString: public ResourceType {
       ExternString* h_str = new ExternString<ResourceType, TypeName>(isolate,
                                                                      data,
                                                                      length);
-      Local<String> str = String::NewExternal(isolate, h_str);
+      MaybeLocal<String> str = String::NewExternal(isolate, h_str);
       isolate->AdjustAmountOfExternalAllocatedMemory(h_str->byte_length());
 
-      return scope.Escape(str);
+      if (str.IsEmpty()) {
+        delete h_str;
+        return Local<String>();
+      }
```

Comment:
@skomski Thanks for the test fix. That's actually what I expected, but I ignored whether the value was intercepted in transit. So my comment about handling this case still stands. Don't want to see aborts on devices that have little memory (e.g. raspberry pi).